hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/lib/MutableQuantiles.java

```diff
@@ -162,7 +206,7 @@ public synchronized void setEstimator(QuantileEstimator quantileEstimator) {
    * Runnable used to periodically roll over the internal
    * {@link SampleQuantiles} every interval.
    */
-  private static class RolloverSample implements Runnable {
+  static class RolloverSample implements Runnable {
```

Comment:
VisibleForTesting?

Reply:
The scope is widened for the class to be accessible from subclass `InversePercentile` and not for Test purposes, so not adding the annotation.